package.json

```diff
@@ -27,7 +27,7 @@
     "localforage": "1.5.6",
     "localforage-observable": "1.4.0",
     "marked": "mattermost/marked#802e981ade71149a497cbe79d12b8a3f82f7657e",
-    "mattermost-redux": "mattermost/mattermost-redux",
+    "mattermost-redux": "mattermost/mattermost-redux#abc-92-no-emoji",
```

Comment:
Is this supposed to be changed here?

Reply:
It was while we were testing, I'll update it back to normal